op-guide/docker-compose.md

```diff
@@ -33,19 +34,25 @@ Make sure you have installed the following items on your machine:
     docker-compose up -d
     ```
 
-3. Access the cluster.
+3. Enable the MySQL Server service and access the cluster.
 
     ```bash
     mysql -h 127.0.0.1 -P 4000 -u root
     ```
 
-    Access the Grafana monitoring interface:
+## Monitor the cluster 
+
+After your machine successfully connects to MySQL Server on `127.0.0.1`, you can monitor real-time activities in the TiDB cluster with:
+
+1. the [Grafana monitoring interface](https://pingcap.com/docs/op-guide/monitor-overview/#about-grafana-in-tidb):
```

Comment:
the -> The
https://pingcap.com/docs/op-guide/monitor-overview/#about-grafana-in-tidb -> monitor-overview.md/#about-grafana-in-tidb